StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethods/SavedPaymentMethodsViewController.swift

```diff
@@ -120,7 +120,7 @@ class SavedPaymentMethodsViewController: UIViewController {
         self.isApplePayEnabled = isApplePayEnabled
         self.spmsCompletion = spmsCompletion
         self.delegate = delegate
-        if savedPaymentMethods.isEmpty {
+        if savedPaymentMethods.isEmpty && !isApplePayEnabled {
```

Comment:
Great catch, but this means that we need to change line 288:

```
                    return savedPaymentMethods.isEmpty && !isApplePayEnabled ? .close(showAdditionalButton: false) : .back
```

Reply:
Done!